examples/with-react-query/README.md

```diff
@@ -0,0 +1,3 @@
+# React Query example
+
+This example have been moved here: [React Query](https://github.com/tannerlinsley/react-query/tree/master/examples/nextjs)
```

Comment:
It would be great if we could have a similar example here, so when you run `create-next-app -e with-react-query`, it works 😄

Reply:
> It would be great if we could have a similar example here, so when you run `create-next-app -e with-react-query`, it works 😄

Will do, thought an external link will be enough, as I saw on the material-ui example

Reply:
hi @leerob ,
I've updated the the pull request